packages/core/src/contracts/tcr/eip20.ts

```diff
@@ -52,7 +68,7 @@ export class EIP20 extends BaseWrapper<EIP20Contract> {
 
   /**
    * Check the token balance of an address
-   * @param address address to check balance of
+   * @param tokenOwner address to check balance of (defaults to current user)
```

Comment:
Should it default to current user or the multisig address.

Reply:
Multisig, if multisig instance - will update.